REQUIRE

```diff
@@ -1 +1,2 @@
 julia 0.6
+StaticArrays
```

Comment:
Shouldn't the version number be specified? Looking at the history of StaticArrays, it looks like there has been some breaking changes in the past.

Reply:
It would take some effort to figure out which version we actually need. So better to wait until someone finds an error in my opinion.

Reply:
Probably v0.5.1. I've had METADATA PRs rejected for not specifying.

Reply:
Ok, sure, I'll go with that